### PR TITLE
Feature/input plugins docs api

### PR DESCRIPTION
**Pull request by AlexTeusz**
Added input plugins documentations to examples, api-reference and README

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -115,6 +115,53 @@ if (!window.cognigyWebchatMessagePlugins) {
...
+            display: 'flex',
+            flexDirection: 'row'
+        }}>
+            <input {...inputProps} />
+            <IconButton
+                style={{
+                    borderTop: '1px solid black',
+                    borderRadius: 0
+                }}
+                onClick={() => {}}
```
Comment (pedily) [inline docs/examples.md]: dat macht nix :D

File: docs/api-reference.md [hunk 2]
```diff
@@ -9,6 +9,17 @@ If set to a string, it will match based on whether a message's `message.data._pl
 Using `message.data._plugin.type` to match the plugin is a convention we try to push (but not enforce).
 It is therefore also possible to provice an own matching function instead of a string, which will get a message as an argument and should return `true` if this plugin should be used, otherwise `false`.
 
+#### `type: string & rule: () => boolean`
```
Comment (pedily) [inline docs/api-reference.md:12]: this does not belong within the "message plugin" documentation. it should be a separate section. 

# Plugins
## Message Plugins
## Input Plugins

File: docs/examples.md (hunk 1 above)
Comment (AlexTeusz) [reply docs/examples.md]: done

File: docs/api-reference.md (hunk 2 above)
Comment (AlexTeusz) [reply docs/api-reference.md]: done

**Pull request merged**